ddtrace/internal/telemetry/writer.py

```diff
@@ -74,6 +88,9 @@ def get_headers(self, request):
         headers = self._headers.copy()
         headers["DD-Telemetry-Debug-Enabled"] = request["debug"]
         headers["DD-Telemetry-Request-Type"] = request["request_type"]
+        headers["DD-Agent-Hostname"] = get_host_info()["hostname"]
```

Comment:
api version is also defined in the request body, it might make sense to keep both values consistent. 

```suggestion
        headers["DD-Telemetry-API-Version""] = request["api_version"]
        headers["DD-Agent-Hostname"] = get_host_info()["hostname"]
```

Reply:
:ok_hand: